src/v/config/configuration.h

```diff
@@ -238,6 +238,11 @@ struct configuration final : public config_store {
     property<size_t> cloud_storage_cache_size;
     property<std::chrono::milliseconds> cloud_storage_cache_check_interval_ms;
 
+    // SI prefetch control
+    property<bool> cloud_storage_prefetch_enable;
```

Comment:
I think it's better to have a 'enable' flag because you might want to enable prefetch without specifying anything else (using defaults for other two parameters).